src/box/lua/load_cfg.lua

```diff
@@ -423,6 +399,17 @@ local dynamic_cfg_modules = {
             flightrec_requests_max_res_size = true,
         },
     },
+    log = {
+        cfg = log.internal.box_api_cfg,
+        settings = log.cfg,
+        skip_at_load = true,
+        options = {
+            log = 'log',
+            log_level = 'level',
+            log_format = 'format',
+            log_nonblock = 'nonblock',
```

Comment:
Please remove the name mapping, because it's inconsistent with other modules and complicates the API. The `box_api.cfg` function should do the mapping by itself.

Reply:
Fixed.